docs/book/component-gallery/experiment-trackers/experiment-trackers.md

```diff
@@ -75,6 +75,9 @@ independently of ZenML
 * finally, you can access the Experiment Tracker UI to browse and visualize the
 information logged during your pipeline runs
 
+Note: the Expirement Tracker will declare run as failed if the pipeline step
```

Comment:
I'd personally remove this note (not crucial information) as it doesn't apply to all the experiment trackers (neptune)